tests/integration_test/test_integration_app.py

```diff
@@ -20,50 +20,56 @@ def test_save_to_netcdf():  # noqa 103
     user_key = os.environ.get("EUMETSAT_USER_KEY")
     user_secret = os.environ.get("EUMETSAT_USER_SECRET")
     with tempfile.TemporaryDirectory() as tmpdirname:
-        response = runner.invoke(
-            run,
-            [
-                "--api-key",
-                user_key,
-                "--api-secret",
-                user_secret,
-                "--save-dir",
-                tmpdirname,
-                "--use-rescaler",
-                False,
-                "--start-time",
-                datetime.datetime.utcnow().isoformat(),
-                "--maximum-n-datasets",
-                1,
-            ],
-            catch_exceptions=False,
-        )
-        assert response.exit_code == 0, response.exception
+
+        run(api_key=user_key,
+            api_secret=user_secret,
+            save_dir=tmpdirname,
+            use_rescaler=False,
+            start_time=datetime.datetime.utcnow().isoformat(),
+            history="30 minutes",
+            maximum_n_datasets=1)
+
+        native_files = list(glob.glob(os.path.join(tmpdirname, "*.zarr.zip")))
+        assert len(native_files) > 0
+
+
+@freeze_time("2024-08-28 12:00:00")  # Date with IODC
+def test_iodc():  # noqa 103
+
+    from satip import utils
+    utils.setupLogging()
+
+    user_key = os.environ.get("EUMETSAT_USER_KEY")
+    user_secret = os.environ.get("EUMETSAT_USER_SECRET")
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Does "/tmp" get mapped out as a volume in the container? Just conscious that a lot of data could get dumped into it

Reply:
yea, im no sure where it goes, but it gets deleted after the test